### DISCUSSION TITLE
Support Deno 1.46

### DISCUSSION THREAD

**Discussion by Mr-Pepe** [supabase/supabase#33055] [Feature Requests]
# Chore

## Describe the chore

Deno 1.46 is the [last 1.x release](https://deno.com/blog/v1.46) of Deno, was published in August, and should be supported by Supabase functions.

## Additional context

Deno 1.46 added support for [Http2Session.socket](https://github.com/denoland/deno/pull/24786), [enabling the use](https://github.com/denoland/deno/issues/20903) of Google Cloud API client libraries. 

It is currently not possible to use some of those libraries in Supabase functions.

**Comment by Hallidayo** [comment]
Hi everyone, due to inactivity on this issue I've moved the issue over to discussions/enhancements.

**Comment by pRizz** [comment]
Any update here? Deno is now at version 2.1.9. I am also running into this exact same Http2Socket issue.

Edit: I made an attempt at updating Deno to 1.46.3 (https://github.com/supabase/edge-runtime/pull/488), but there are some outstanding issues. Help is aappreciated!
